src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({
+  person: { name, age, isMarried, sex, partnerName },
+}) => {
+  // const sexCheck = () => {
+  //   return sex === 'm' ? 'wife' : 'husband';
+  // };
+  const marriedMale = 'wife';
+  const marriedFemale = 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
+      <p className="Person__partner">
+        {(isMarried &&
+          `${partnerName} is my ${sex === 'm' ? marriedMale : marriedFemale}`) ||
+          `I am not married`}
```

Comment:
This condition looks too complicated, different logical operators and ternary at the same time, try to simplify